client_test/container_app_test.py

```diff
@@ -58,13 +58,13 @@ def get_stub():
 
     stub = get_stub()
 
-    # No container is running
-    assert not stub.is_inside()
-    assert not stub.is_inside(image_1)
-    assert not stub.is_inside(image_2)
-
     # Run container
     async with stub.run(client=aio_client) as app:
+        # We're not inside the container (yet)
+        assert not stub.is_inside()
```

Comment:
super minor but might as well keep both cases around 🤷

Reply:
yeah probably. my rationale was that "there is an app running on this stub" is a strict subset of "this stub exists" so it does increase the specificity. but maybe that's maybe making an implicit assumption.